website/content/en/docs/building-operators/ansible/reference/internal_metrics.md

```diff
@@ -0,0 +1,16 @@
+The Ansible Operator comes with three internal metrics that provide an insight to the frequency and time of operator reconciliations. These metrics can be
+scraped by a Prometheus instance or any other openmetrics system. To publish operator metrics and scrape them with an openmetrics system such as Prometheus, view 
+[Kubebuilder documentation](https://book.kubebuilder.io/reference/metrics.html) on publishing metrics.
+
+The default metrics recorded in Operator-SDK are collected in a [histogram](https://prometheus.io/docs/practices/histograms/).
```

Comment:
When referring to the command you use `operator-sdk` when referring to the project name use "Operator SDK"

```suggestion
The default metrics recorded in Operator SDK are collected in a [histogram](https://prometheus.io/docs/practices/histograms/).
```